app/views/editor/rich.html.erb

```diff
@@ -62,12 +62,12 @@
       </div>
 
       <div class="ple-module-content col-lg-9">
-        <input 
-          class="form-control input-lg" 
-          type="text" 
+        <input
+          class="form-control input-lg"
+          type="text"
           id="title-input"
-          placeholder="Title" 
-          value="<%= if @node then @node.title else params[:title] end %>" 
+          placeholder="Title"
+          value="<%= if @node then @node.title else params[:title] end %>"
         />
```

Comment:
I'm not sure why it made all these changes to the rest of the lines?

Reply:
Maybe its the text editor... I use vim and whenever I open a file with vim all the trailing white spaces are auto deleted...strange that the travis is complaining.. Restarting it... Thanks so much @nstjean and thanks a lot @pianeta051 for working on this